udfs/README.md

```diff
@@ -45,6 +44,26 @@ or
 
 `CREATE OR REPLACE FUNCTION project_id.your_dataset.function_name() . . .`
 
+### Using JavaScript UDFs
+
+When creating [JavaScript UDFs](https://cloud.google.com/bigquery/docs/reference/standard-sql/user-defined-functions#javascript-udf-structure) in your dataset, you need both to create 
+the UDF and upload the javascript library to a publicly available Google Storage Bucket. 
+
+In the following example, we show how to create the Levenshtein UDF function, that uses a JS
+library, in your dataset.
+
+1. We [obtain](https://www.npmjs.com/package/js-levenshtein) the latest version of the
+`js-levenshtein` function.
+2. We copy the library to our bucket:
+`gsutil cp js-levenshtein-v1.1.6.js gs://your-bucket`
```

Comment:
Let's make this an easy 1 step by having them copy the JavaScript webpack-generated bundle from the publicly hosted GCS bigquery-utils bucket: gs://bqutil-lib/bq_js_libs/

`gsutil cp gs://bqutil-lib/bq_js_libs/js-levenshtein-v1.1.6.js gs://your-bucket`

Reply:
Done